### ISSUE TITLE
Performance and Accessibility Testing

### ISSUE THREAD

**Issue by vijaylaxmipb** [vijaylaxmipb/Inkspire-Insights#17]
As a **developer**, I want to **ensure that application meets performance and accessibility standards so that provides a seamless experience for all users and performs optimally.**

*** Acceptance criteria

- Page load times should be measured and documented using tool Lighthouse.

- Accessibility audits should be conducted using tools Wave or Lighthouse.

- Results and if any issues identified included in README.md.
